### PR TITLE
moonlight: add core info for switch running lakka

### PR DESCRIPTION
**Pull request by plaidman**

### CODE REVIEW

File: dist/info/moonlight_libretro.info [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+display_version = "1.0.3"
+
+# Hardware Information - Information about the hardware the core supports (when applicable)
+# manufacturer = "nVidia"
+# systemname = "Moonlight"
+
+# Libretro Features - The libretro API features the core supports. Useful for sorting cores
+# savestate = "false"
+# cheats = "false"
+# supports_no_game = "true"
```
Comment (RobLoach) [inline dist/info/moonlight_libretro.info:18]:
``` suggestion
supports_no_game = "true"
```

File: dist/info/moonlight_libretro.info [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# hw_render = "true"
+# is_experimental = "true"
+
+# Descriptive text, useful for tooltips, etc.
+# description = "A port of the nVidia streaming client Moonlight. Currently only tested using Lakka running on Nintendo Switch."
```
Comment (RobLoach) [inline dist/info/moonlight_libretro.info:23]:
``` suggestion
# Hardware Information - Information about the hardware the core supports (when applicable)
manufacturer = "nVidia"
systemname = "Moonlight"

# Libretro Features - The libretro API features the core supports. Useful for sorting cores
savestate = "false"
cheats = "false"
supports_no_game = "true"
hw_render = "true"
is_experimental = "true"

# Descriptive text, useful for tooltips, etc.      
description = "A port of the nVidia streaming client Moonlight. Currently only tested using Lakka running on Nintendo Switch."
```

Comment (plaidman) [reply dist/info/moonlight_libretro.info]: good call. updated.

File: dist/info/moonlight_libretro.info [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# Software Information - Information about the core software itself
+display_name = "Moonlight"
+categories = "Streaming client"
+authors = "Rock88"
+corename = "Moonlight"
+license = "GPLv3"
+display_version = "1.0.3"
+
+# Hardware Information - Information about the hardware the core supports (when applicable)
+manufacturer = "nVidia"
```
Comment (RobLoach) [inline dist/info/moonlight_libretro.info:12]: It's branded as NVIDIA in most places on their website :shrug: 

``` suggestion
manufacturer = "NVIDIA"
```

Comment (plaidman) [reply dist/info/moonlight_libretro.info]: oh yea missed that from your first comment. updated again.

File: dist/info/moonlight_libretro.info [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## All data is optional, but helps improve user experience.
+
```
Comment (RobLoach) [inline dist/info/moonlight_libretro.info:2]: Don't think we need this header here :wink: 
``` suggestion
```

**Review by RobLoach — APPROVED** [review]
I haven't tested the Moonlight core, so I'm not sure if it's in a workable state, but doesn't hurt to have the .info file there. Might want to check with @Rock88 to see how he feels about pushing this forwards. Thanks!

File: dist/info/moonlight_libretro.info (hunk 4 above)
Comment (plaidman) [reply dist/info/moonlight_libretro.info]: updated.

**Comment by plaidman** [comment]
We have reached out to rock88 a few times but so far he's been unresponsive. I think he's working on a proper switch port now, and may have abandoned the lakka port.

**Comment by plaidman** [comment]
Though we have tested it in its current state and it works great!

**Pull request merged**